colaraz/lms/static/js/discussion/colaraz_discussion_filters.js

```diff
@@ -109,7 +110,7 @@ function loadMoreAllTopics(element) {
 
 function loadMoreButtonAllTopics(next_url) {
     return `<li class="forum-nav-thread" style="user-select: auto;">
-                <a id="cz-loadmore" href="#" class="forum-nav-thread-link" next-url="${next_url}" onclick="loadMoreAllTopics(this)">load more</a>
+                <a id="cz-loadmore" href="#" class="forum-nav-thread-link " next-url="${next_url}" onclick="loadMoreAllTopics(this)">load more</a>
```

Comment:
Why are we adding space in class? Is there any specific reason behind this?

Reply:
typo resolved.